dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/SchedulerServiceImpl.java

```diff
@@ -401,7 +403,7 @@ public Result querySchedule(User loginUser, long projectCode, long processDefine
         Project project = projectMapper.queryByCode(projectCode);
 
         // check project auth
-        boolean hasProjectAndPerm = projectService.hasProjectAndPerm(loginUser, project, result);
+        boolean hasProjectAndPerm = projectService.hasProjectAndPerm(loginUser, project, result, PROJECT);
```

Comment:
PTAL @WangJPLeo

Reply:
ok, update and keep the checkDesc method. @caishunfeng

Reply:
PTAL @WangJPLeo

Reply:
No problem here, LGTM.